fvm/blueprints/fees.go

```diff
@@ -1,29 +1,48 @@
 package blueprints
 
 import (
+	_ "embed"
 	"encoding/hex"
 	"fmt"
 
 	"github.com/onflow/cadence"
 	jsoncdc "github.com/onflow/cadence/encoding/json"
+	"github.com/onflow/cadence/runtime/common"
 	"github.com/onflow/flow-core-contracts/lib/go/contracts"
 
 	"github.com/onflow/flow-go/model/flow"
 )
 
-const TransactionExecutionParametersPathDomain = "storage"
-const TransactionFeesExecutionEffortWeightsPathIdentifier = "executionEffortWeights"
-const TransactionFeesExecutionMemoryWeightsPathIdentifier = "executionMemoryWeights"
-const TransactionFeesExecutionMemoryLimitPathIdentifier = "executionMemoryLimit"
-
-const deployTxFeesTransactionTemplate = `
-transaction {
-  prepare(flowFeesAccount: AuthAccount, serviceAccount: AuthAccount) {
-    let adminAccount = serviceAccount
-    flowFeesAccount.contracts.add(name: "FlowFees", code: "%s".decodeHex(), adminAccount: adminAccount)
-  }
+var TransactionFeesExecutionEffortWeightsPath = cadence.Path{
+	Domain:     common.PathDomainStorage.Identifier(),
+	Identifier: "executionEffortWeights",
+}
+var TransactionFeesExecutionMemoryWeightsPath = cadence.Path{
+	Domain:     common.PathDomainStorage.Identifier(),
+	Identifier: "executionMemoryWeights",
 }
-`
+var TransactionFeesExecutionMemoryLimitPath = cadence.Path{
+	Domain:     common.PathDomainStorage.Identifier(),
+	Identifier: "executionMemoryLimit",
+}
+
+//go:embed scripts/deployTxFeesTransactionTemplate.cdc
```

Comment:
what is the main benefit of using go:embed here, I found it making readability a bit worse now that you have to look up two places. 

I'm also worried we are using cdc files that are not a working version of cdc and are templates that requires replacement, maybe we could define a new file extension for cadence templates ?